pkg/command/cmd_executor.go

```diff
@@ -97,7 +96,8 @@ func (p *DoFunctionProvider) getMatchingAction(comment *gogh.IssueCommentEvent)
 
 // Execute triggers the given DoFunctions (when all checks are fulfilled) for the given pr comment
 func (e *CmdExecutor) Execute(client ghclient.Client, log log.Logger, comment *gogh.IssueCommentEvent) error {
-	if e.Command != strings.TrimSpace(*comment.Comment.Body) {
+	body := strings.TrimSpace(*comment.Comment.Body)
+	if prefix := strings.Split(body, " ")[0]; e.Command != body && prefix != e.Command {
```

Comment:
The reason why I am checking both because for `/run test-keeper` we need to check only first parameter. But if you see for `/ok-without-tests` it has only body, it doesn't have any parameter. So here if we try to check only first string then suppose any user trigger `\ok-without-tests foo` then also it'll execute which should not be the case.